tcms/telemetry/templates/telemetry/testing/breakdown.html

```diff
@@ -40,14 +66,14 @@
             </div>
 
             <div class="bar-legend">
-                <div class="bar-legend" style="display: flex; justify-content: space-around">
+                <div class="bar-legend">
```

Comment:
Duplicate div tag with the same class breaks alignment for child widgets. Remove this div (leaving the outer one)  and it should be good to go.